lib/module.js

```diff
@@ -42,6 +42,13 @@ export default async function (moduleOptions) {
     head[serializeProp][options.hid] = ['innerHTML']
   })
 
+  this.nuxt.hook('vue-renderer:ssr:csp', (cspScriptSrcHashes) => {
+    const { csp } = this.options.render
+    const hash = crypto.createHash(csp.hashAlgorithm)
```

Comment:
You're missing an `import crypto from 'crypto'` to be able to use "crypto" here.